modules/govuk_elasticsearch/manifests/plugins.pp

```diff
@@ -33,11 +33,16 @@
 
   }
   # If the version is 2.4, then install the plugin the 2.4 way
-  elsif versioncmp($elasticsearch_version, '2.4') == 0 {
+  elsif versioncmp($elasticsearch_version, '2.4.6') == 0 {
```

Comment:
I thought the `versioncmp` function dealt with minor versions but I guess not. Might be safer to do a regex upon the versions (ie if we _did_ patch I wouldn't want Puppet to suddenly trash stuff because of this conditional)